packages/rx_bloc_cli/mason_templates/bricks/lib_dev_menu/__brick__/lib/base/extensions/dio_extension.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:dio/dio.dart';
+import 'package:dio/io.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../../base/data_sources/remote/http_clients/api_http_client.dart';
+import '../../lib_auth/data_sources/remote/interceptors/auth_interceptor.dart';
+
+extension DioFactoryX on Dio {
+  static String proxy = '';
+
+  static Dio refreshTokenInstance = newInstance();
```

Comment:
I believe this needs to be address properly @kostadin-damyanov-prime can you help here please ?